src/main/java/de/unistuttgart/overworldbackend/service/CourseService.java

```diff
@@ -453,27 +444,21 @@ private MinigameTask cloneFinitequiz(
     private MinigameTask cloneTowercrush(
```

Comment:
I am fairly certain that it is possible to refactor all these single methods into one common method…